### DISCUSSION TITLE
Tile Destruction Issues

### DISCUSSION THREAD

**Discussion by STBlitzkrieg** [Pryaxis/TShock#2227] [Q&A]
I've been working on a TShock server lately and we were going to do a stress test on our new GSP. We loaded up the Vanillafy plugin, ran it on the server, and all joined. We primed our Celebration MKIIs and Mini Nuke IIs and...

![test2](https://user-images.githubusercontent.com/16367090/107863871-573cb480-6e4f-11eb-98d7-4585058f9f5d.png)

It seems tile destruction protection is still enabled. Whatever, I hopped in the config and set any offending settings that could have possibly had anything to do with tilekill / thresholds. Still no dice. I go into the Owner and Vanilla groups in the database and add tshock.ignore.removetile to the group permissions.

Still nothing. I'm very confused at this point. Even as the owner, with spawn protection off, Vanillafy running, and config and database edits, I'm still getting an enormous number of these bouncer messages in chat. Anyone know a solution?

**Comment by STBlitzkrieg** [comment]
Bumping this as it's been over a month. Hoping someone can shed some light on this.

**Comment by Kojirremer** [comment]
Is is just the messages or someone is getting disabled? Because the messages are most likely due to "_DebugLogs_" on the **config.json** file being set to true.

**Comment by moisterrific** [comment]
I believe this issue has been fixed as of the latest version. I think this was due to Celebration Mk2 not being properly added to TSAPI.
